ci-operator/step-registry/optional-operators/ci/run-bundle/optional-operators-ci-run-bundle-ref.yaml

```diff
@@ -0,0 +1,36 @@
+ref:
+  as: optional-operators-ci-run-bundle
+  from_image:
+    namespace: ci
+    name: cvp-operator-scorecard
+    tag: v1
+  commands: optional-operators-ci-run-bundle-commands.sh
+  resources:
+    requests:
+      cpu: 300m
+      memory: 300Mi
+  dependencies:
+  - name: bundle-image
+    env: OO_BUNDLE_IMG
+  env:
+  - name: OO_INSTALL_NAMESPACE
+    default: "!create"
+    documentation: |
+      This namespace must already exist in the cluster.
```

Comment:
This is not useful in a context of a CI job - that's the reason why the other step supported the special `!create` value. IMO, this step needs to support it too.

Reply:
Are you suggesting that the step should support a namespace doesn't already exist in the cluster?

Reply:
Yes. Exactly like the other step does. It's a common testing scenario: the operator needs to be installed in *some* namespace, but the CI job author does not care which one, so the other step implemented the `!create` special value for this.

Reply:
The step now handles creation of namespace, similar to the `ci-subscribe` operator.